libs/libxs/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2016-2019  Moddable Tech, Inc.
+#
+#   This file is part of the Moddable SDK Tools.
+# 
+#   The Moddable SDK Tools is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+# 
+#   The Moddable SDK Tools is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+# 
+#   You should have received a copy of the GNU General Public License
+#   along with the Moddable SDK Tools.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libxs
+PKG_VERSION:=1.0
```

Comment:
The linked repository makes no mention like this. Use PKG_SOURCE_DATE instead. Use this Makefile as a guide: https://github.com/openwrt/packages/blob/master/kernel/cifsd/Makefile